packages/shared/src/components/search/SearchBarSuggestion.tsx

```diff
@@ -14,7 +14,7 @@ export const SearchBarSuggestion = <TagName extends AllowedTags>({
 }: ButtonProps<TagName>): ReactElement => {
   return (
     <Button
-      spanClassName="w-fit py-2 flex-shrink"
+      spanClassName="w-fit my-2 flex-shrink laptop:line-clamp-2"
```

Comment:
What actually happens here if you did line-clamp-1?
Just wondering

Reply:
It would only keep 1 line.

Reply:
Hmm but the one-line with ellipsis wouldn't work?
It's more for a future improvement. I might take a look at it later, fun CSS Challenge

Reply:
It should work. It would look like below. Not sure if that is what you are envisioning, but I agree, it would be a cool challenge.

![image](https://github.com/dailydotdev/apps/assets/13744167/f21369ee-694a-420f-8574-3cc1e1e7be05)

Reply:
Yeah did you share this as a alternative with design?
I'd be cool to find out how you could make them dynamically flex-grow available space and just hide the ones that don't fit 😅

Reply:
Yes, the single line is what I initially shared. It was mostly through JavaScript, calculating how many pills would exceed the container's width and then deciding how much would fit.